pytest-selenium/regions/my_highlights.py

```diff
@@ -845,6 +860,28 @@ def toggle_menu_visible(self):
                 except TimeoutException:
                     return False
 
+            @property
+            def highlight_edit_boxes(self) -> List[WebElement]:
+                """Return the list of highlight edit box elements in the modal.
+
+                :return: the list of highlight edit box elements
+                :rtype: list(WebElement)
+
+                """
+                return self.find_elements(*self._highlight_edit_box_locator)
+
+            @property
+            def highlight_edit_box_open(self):
+                """Search for the open (displayed) highlight edit box."""
+                for _ in range(2):
+                    for box in self.highlight_edit_boxes:
+                        sleep(0.1)
+                        display = self.driver.execute_script(
+                            COMPUTED_STYLES.format(field=".display"), box
+                        )
+                        if display != "none":
+                            return True
```

Comment:
this should really have a `return False` at the end (even with the first `for` loop) to make it clear that this is a boolean property even though `None` == `False` when evaluated